catenax-ng/gh-org-checks
# Feat/replace angular with backend render

## Changes

- replace angular frontend with backend html rendering
- add `static` dir with css content
- add `template` dir with go template for main page
- adjust `Dockerfile` to include files and remove frontend build part
- move the design closer to the [app dashboard](https://github.com/eclipse-tractusx/app-dashboard)
- add [simple-datatables](https://github.com/fiduswriter/simple-datatables/blob/main/docs/demos/3-cdn/index.html) component to make table searchable, sortable
- remove angular code

## How to test locally

1. Clone repository on current branch
2. Run `npm i`
3. Run `npm start`
4. Wait for scan to complete (look for debug console message: `check completed!`
5. Open browser in `http://localhost:8000`
